update-lb4-docs.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+const srcDocs = path.resolve(
+  __dirname,
+  'node_modules',
+  '@loopback',
+  'docs',
+  'site',
+);
```

Comment:
It can be simplified as  `const srcDocs = path.resolve(__dirname, 'node_modules/@loopback/docs/site');`.